pykern/pkcli/__init__.py

```diff
@@ -66,6 +66,41 @@ def command_error(fmt, *args, **kwargs):
     raise CommandError(fmt.format(*args, **kwargs))
 
 
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
+                      argparse.RawDescriptionHelpFormatter):
+    def _expand_help(self, action):
+        r = super()._expand_help(action)
+        return r.split('\n')[0]
+
+class CustomParser(argparse.ArgumentParser):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.program = { key: kwargs[key] for key in kwargs }
```

Comment:
You should be able to just assign kwargs or PKDict(kwargs). If you are concerned use kwargs.copy() or really concerned: copy.deepcopy(kwargs).